src/test/java/com/hocs/test/pages/create_case/SuccessfulCaseCreation.java

```diff
@@ -32,4 +50,24 @@ public String getCaseReference() {
         return caseReference;
     }
 
+    // Call session variable and set it to linkText of a findElement.by
+
+    public void clickSessionVariableViaLinkText() {
```

Comment:
change "SessionVariable" to what you are actually clicking. e.g caseReferenceLink